### PR TITLE
Add Call for Proposal page

### PR DESCRIPTION
**Pull request by bhuvana-s**
Please merge the changed made to include **Call for Proposal** page.

### CODE REVIEW

File: call_for_proposal.html [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+	<link rel="apple-touch-icon" sizes="114x114" href="https://a0.awsstatic.com/main/images/site/touch-icon-iphone-114-smile.png"
+	/>
+	<link rel="apple-touch-icon" sizes="144x144" href="https://a0.awsstatic.com/main/images/site/touch-icon-ipad-144-smile.png"
+	/>
+
+	<link rel="canonical" href="https://communityday.awsugblr.in/" />
+
+	<meta property="og:url" content="https://communityday.awsugblr.in" />
+	<meta property="og:type" content="website" />
+	<meta property="og:title" content="AWS Community Day 2018 " />
```
Comment (SathyaBhat) [inline call_for_proposal.html]: Can we change the title to AWS Community Day 2018 Bengaluru. We'll add for the other page later

File: call_for_proposal.html [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+  <meta property="og:description" content="AWS Community Day is a community-organized event featuring technical discussions led by expert AWS users and community leaders. AWS Community Day is being organized by AWS Users Group - Bangalore. This group is for anyone interested in Amazon Web Services, Cloud Computing, AWS design implementation and servicing, EC2, S3, High Scalability Computing, Real Life uses of AWS and cloud servers, SaaS, IAAS, PAAS and many more" />
+	<meta property="og:image" content="https://s3.ap-south-1.amazonaws.com/aws-community-blr/community-day-logo-sq.png" />
+	<meta property="og:image:secure_url" content="https://s3.ap-south-1.amazonaws.com/aws-community-blr/community-day-logo-sq.png"
+	/>
+	<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+	<title>AWS Community Day 2018 Bengaluru - Call for Proposal</title>
```
Comment (SathyaBhat) [inline call_for_proposal.html]: please change the title to "Call for Proposals - AWS Community Day 2018 Bengaluru"

File: call_for_proposal.html [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+
+	<!-- section proposal Start	-->
+	<section id="section-proposal" class="section-padding ">
+		<div class="container">
+			<div class="row">
+
+				<div class="col-md-12 col-sm-12">
+					<div class="about-intro-1">
+					<p>					
+						We’re excited to open the Call for Proposals for AWS Community Day 2018!
```
Comment (SathyaBhat) [inline call_for_proposal.html]: AWS Community Day 2018 Bengaluru

File: call_for_proposal.html (hunk 1 above)
Comment (jeevandongre) [reply call_for_proposal.html]: Find and Replace Bangalore to Bengaluru.

File: index.html [hunk 4]
```diff
@@ -70,7 +69,7 @@
 	</style>
 
 
-	<script src="https://platform-api.sharethis.com/js/sharethis.js#property=5b63095a78eb8b00113e362f&product=inline-share-buttons"></script>
+
```
Comment (SathyaBhat) [inline index.html]: please add this, this gets us the sharing buttons

File: index.html [hunk 5]
```diff
@@ -341,26 +334,36 @@ <h2>Supporting Organizations</h2>
 		</div>
 	</section>-->
 
-    
-	<div class="sharethis-inline-share-buttons"></div>
-<!-- 
+
```
Comment (SathyaBhat) [inline index.html]: add the inline-share-buttons div, remove the fb one

File: call_for_proposal.html (hunk 3 above)
Comment (bhuvana-s) [reply call_for_proposal.html]: done

**Pull request merged**